README.md

```diff
@@ -19,190 +18,195 @@ A repository to learn Algorithms via contributing in different languages, contri
+  - [JavaScript](sorting/quick_sort/quickSort.js)
+  - [Python](sorting/quick_sort/animatedQuickSort.py)
+- Radix Sort
+  - [C++](sorting/radix_sort/radixSort.cpp)
+- Selection Sort
+  - Ascending
+    - [C - using max element](sorting/selection_sort/ascSelectionSortWithMax.c)
+    - [C - using min element](sorting/selection_sort/ascSelectionSortWithMin.c)
+    - [Java - using max element](sorting/selection_sort/ascSelectionSort.java)
+    - [Go - using min element](sorting/selection_sort/selectionSort.go)
+    - [C++](sorting/selection_sort/ascSelectionSort.cpp)
+  - Descending
+    - [C - using max element](sorting/selection_sort/descSelectionSortUsingMax.c)
+    - [C - using min element](sorting/selection_sort/descSelectionSortUsingMin.c)
+    - [Java - using min element](sorting/selection_sort/descSelectionSort.java)
+- Shell Sort
+  - Ascending
+    - [Java](sorting/shell_sort/ascShellSort.java)
+  - Descending
+    - [Java](sorting/shell_sort/descShellSort.java)
+- Bucket Sort
+  - Ascending
+    - [C++](sorting/bucket_sort/bucketSort.cpp)
+
+---
 
 ### strings
 
-* Anagram
-  * [C](strings/anagram/anagram.c)
-  * [C++](strings/anagram/anagram.cpp)
-  * [Java](strings/anagram/anagram.java)
-  * [Python](strings/anagram/anagram.py)
-* Palindrome
-  * [C](strings/palindrome/palindrome.c)
-  * [C++](strings/palindrome/palindrome.cpp)
-  * [Java](strings/palindrome/palindrome.java)
-  * [Python](strings/palindrome/palindrome.py)
-* Pattern Searching
-  * KMP
-    * [C++](strings/pattern_searching/kmp/kmp.cpp)
-  * Z
-    * [C++](strings/pattern_searching/z/z.cpp)
-* Print KPC
-  * [Python](strings/print_kpc/printKpc.ipynb)
-* Reverse a String
-  * [C](strings/reverse_a_string/reverseString.c)
-  * [C++](strings/reverse_a_string/reverseString.cpp)
-  * [Java](strings/reverse_a_string/reverseString.java)
-  * [Python](strings/reverse_a_string/reverseString.py)
-  * [Ruby](strings/reverse_a_string/reverseString.rb)
-* Subsequence
-  * [Python](strings/subsequence/subsequence.ipynb)
-
-***
+- Anagram
+  - [C](strings/anagram/anagram.c)
+  - [C++](strings/anagram/anagram.cpp)
+  - [Java](strings/anagram/anagram.java)
+  - [Python](strings/anagram/anagram.py)
+- Palindrome
+  - [C](strings/palindrome/palindrome.c)
+  - [C++](strings/palindrome/palindrome.cpp)
+  - [Java](strings/palindrome/palindrome.java)
+  - [Python](strings/palindrome/palindrome.py)
+- Pattern Searching
+  - KMP
+    - [C++](strings/pattern_searching/kmp/kmp.cpp)
+  - Z
+    - [C++](strings/pattern_searching/z/z.cpp)
+- Print KPC
+  - [Python](strings/print_kpc/printKpc.ipynb)
+- Reverse a String
+  - [C](strings/reverse_a_string/reverseString.c)
+  - [C++](strings/reverse_a_string/reverseString.cpp)
+  - [Java](strings/reverse_a_string/reverseString.java)
+  - [Python](strings/reverse_a_string/reverseString.py)
+  - [Ruby](strings/reverse_a_string/reverseString.rb)
+- Subsequence
+  - [Python](strings/subsequence/subsequence.ipynb)
+
+---
```

Comment:
So in this PR all the * is replaced with - character in README I guess some IDE is doing this
So the correct way to update the README i.e.
only update the code file link in the README that you have added
for eg: You have added the Ascending Selection Sort algorithm
Then in that case under sorting heading, under selection sort heading, under ascending heading
you have to put `* [C++](sorting/selection_sort/ascSelectionSort.cpp)` only instead of updating all the README